scripts/hdl/simify_verilog.py

```diff
@@ -21,11 +21,17 @@ def add_ispm(contents: str, filename: str) -> str:
     assert m is not None
 
     orig_str = m.group()
-    new_str = f"{orig_str} initial $readmemh(\"{filename}\", ispm);"
+    new_str = f"""
+initial begin
+  $value$plusargs("ispm=%s", hex_file_name);
+  $readmemh(hex_file_name, ispm);
```

Comment:
?

Reply:
The idea here is that we can now pass the hex file into the emulator using command line arguments. So that the test harness can reuse the same emulator to run different programs with distinct filenames (as opposed to `imem.hex.txt` only). Is this a good way to implement this feature?

Reply:
Isn't that what was already happening?

Reply:
On the master branch, the emulator only recognizes a hex file with the name `imem.hex.txt`. The change above allows it to run different programs by passing in their filenames without having to rename any programs as `imem.hex.txt`.

So if we have three test programs, `a.hex`, `b.hex`, and `c.hex`. To run them, we just need to do
```
flexpret-emulator +ispm=a.hex
flexpret-emulator +ispm=b.hex
flexpret-emulator +ispm=c.hex
```
without having to change any of the hex files to `imem.hex.txt` first.